pages/zh/lb4/Binding.md

```diff
@@ -0,0 +1,192 @@
+---
+lang: zh
+title: 'Binding'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI
+sidebar: lb4_sidebar
+permalink: /doc/zh/lb4/Binding.html
+---
```

Comment:
add one more line after `permalink` line to indicate it's translation. See https://raw.githubusercontent.com/strongloop/loopback.io/gh-pages/pages/zh/lb4/Relations.md as an example. 

```
layout: translation
```